src/app/model/User.ts

```diff
@@ -72,34 +69,6 @@ class User extends BaseModel {
     return this.getCollection("users").where("displayName", ">=", value).limit(limit).get();
   }
 
-  /**
-   * Given a mission object creates a new mission in firestore
-   * returns the new mission id
-   * @param {object} mission
-   * @return {string}
-   */
-  async createMission(mission: MissionInterface): Promise<string> {
```

Comment:
Moved this method to `Mission.ts` as it makes more sense there is actually associated with the correct default data